.travis.yml

```diff
@@ -77,6 +72,9 @@ jobs:
         - cmake --build ringmesh --target package
         - cd $TRAVIS_BUILD_DIR
         - sonar-scanner
+      after_failure:
+        - if [ "$TRAVIS_BRANCH" = "master" ]; then 
```

Comment:
AND "$TRAVIS_EVENT_TYPE" != "pull_request" (cf https://docs.travis-ci.com/user/environment-variables/#Default-Environment-Variables)

Reply:
Yes it was for testing (since it is a PR)